packages/block-editor/src/components/index.native.js

```diff
@@ -25,3 +25,5 @@ export { default as BlockEditorProvider } from './provider';
 // Mobile Editor Related Components
 export { default as BottomSheet } from './mobile/bottom-sheet';
 export { default as Picker } from './mobile/picker';
+
+export { default as BlockInvalidWarning } from './block-list/block-invalid-warning';
```

Comment:
This is temporal while we still have `BlockHolder` in the `gutenberg-mobile` side.